android/examples_demos/Retrofit-with-EventBus/Retrofit_With_EventBus/app/build.gradle

```diff
@@ -19,39 +22,65 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            shrinkResources true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    flavorDimensions "string"
+
+    productFlavors {
+        mock {
+            applicationIdSuffix = ".mock"
+        }
+        prod {
+        }
+    }
+
+    // Remove mockRelease as it's not needed.
+    android.variantFilter { variant ->
+        if (variant.buildType.name.equals('release')
+                && variant.getFlavors().get(0).name.equals('mock')) {
+            variant.setIgnore(true)
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
     //
-    compile supportDependencies.appCompat
+    implementation supportDependencies.appCompat
     //
-    compile supportDependencies.cardView
+    implementation supportDependencies.annotations
     //
-    compile supportDependencies.design
+    implementation supportDependencies.cardView
     //
-    compile supportDependencies.recyclerView
+    implementation supportDependencies.design
     //
-    compile supportDependencies.supportV4
+    implementation supportDependencies.recyclerView
     //
-    compile supportDependencies.supportVectorDrawable
+    implementation supportDependencies.supportV4
     //
-
-    compile 'com.github.bumptech.glide:glide:3.7.0'
-    compile 'com.google.code.gson:gson:2.8.0'
-    compile 'com.jakewharton:butterknife:8.5.1'
-    compile 'com.squareup.okhttp3:logging-interceptor:3.6.0'
-    compile 'com.squareup.retrofit2:converter-gson:2.2.0'
-    compile 'com.squareup.retrofit2:retrofit:2.2.0'
-    compile 'org.greenrobot:eventbus:3.0.0'
-    compile 'com.android.support.constraint:constraint-layout:1.0.2'
-    annotationProcessor 'com.jakewharton:butterknife-compiler:8.5.1'
-
-    compile('com.mikepenz:aboutlibraries:5.9.4@aar') {
+    implementation supportDependencies.supportV13
+    //
+    implementation supportDependencies.supportVectorDrawable
+    //
+    implementation supportDependencies.exifInterface
+    //
+    implementation 'com.github.bumptech.glide:glide:3.8.0'
```

Comment:
You have taken dependencies.gradle separately so I think these all dependencies should also be referenced from that gradle file itself.